tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
-  "include": ["src", "types"],
+  "include": ["src", "types", "src/Button/.test.tsx", "src/ChatCard/.test.tsx", "src/Input/.test.tsx", "src/ChatForm/.test.tsx"],
```

Comment:
is there some regex we can use here instead?

Reply:
I didn't realize that got added. I'm gonna try and take it out - shouldn't be there imo.